supernode/daemon/mgr/scheduler/manager.go

```diff
@@ -195,6 +196,7 @@ func (sm *Manager) getPieceResults(ctx context.Context, taskID, clientID, peerID
 func (sm *Manager) tryGetPID(ctx context.Context, taskID string, pieceNum int, peerIDs []string) (dstPID string) {
 	defer func() {
 		if dstPID == "" {
+			logrus.Warnf("scheduler: failed to try get pid and use super pid as default")
```

Comment:
failed to get pid. remove `try`

Reply:
I think that the `warn` level isn't appropriate because that it's a normal scheduling result at beginning. In addition, the log is lack of key information, such as taskId, pid.

Reply:
> I think that the `warn` level isn't appropriate because that it's a normal scheduling result at beginning. In addition, the log is lack of key information, such as taskId, pid.

Do you have any good ideas for not use `WARN` level?

Reply:
OK. Forget it, I will delete this log.